packages/validate/translations/en-PH.js

```diff
@@ -0,0 +1,11 @@
+import en from './en.js';
+
+export default {
+  ...en,
+  error: {
+    isDate: 'Please enter a valid date (MM/DD/YYYY).',
+  },
+  warning: {
+    isDate: 'Please enter a valid date (MM/DD/YYYY).',
```

Comment:
to clarify: is this a standard date format everywhere in Philippines?

Reply:
Yes, for short format dates like in the datepicker input

Reply:
`Intl.DateTimeFormat` enforces a different standard

```js
new Intl.DateTimeFormat('en-PH').format(new Date('2019/06/30'))
// "30/06/2019"
new Intl.DateTimeFormat('en-GB').format(new Date('2019/06/30'))
// "30/06/2019"
new Intl.DateTimeFormat('en-US').format(new Date('2019/06/30'))
// "6/30/2019"
```

I'm not sure if we can or should deviate from it

Reply:
so the rest of this MR is OK, but this is not good, in fact you'll have same problem we discussed previously when user will enter date in the format of `MM/DD/YYYY` in the `lion-date`, but because of how Intl works the date will be invalid, because for `en-PH` Intl would expect to see `DD/MM/YYYY`

Reply:
looks like you have 2 different format in the country, more context here: https://en.wikipedia.org/wiki/Date_and_time_notation_in_the_Philippines

and they are predefined in locales, e.g.:
- http://www.localeplanet.com/icu/en-PH/index.html => Pattern: dd/MM/y
- http://www.localeplanet.com/icu/fil-PH/index.html => Pattern: M/d/yy

so with our current design you can't have format `M/d/yy` if you want to use English (en-PH)

Reply:
Noted on this, and apologies on our part as we've overlooked such resources. We'll be sticking to DD/MM/YYYY.